code/modules/mining/machine_redemption.dm

```diff
@@ -500,6 +506,36 @@
 							"<span class='notice'>You insert [I] into [src].</span>")
 	return TRUE
 
+/obj/machinery/mineral/ore_redemption/proc/try_refill_storage(mob/living/silicon/robot/robot)
+	. = FALSE
+	if(!istype(robot))
+		return
+	if(!istype(robot.module, /obj/item/robot_module/engineering)) // Should only happen for drones
+		return
+
+	for(var/datum/robot_storage/material/mat_store in robot.module.material_storages)
+		if(mat_store.amount == mat_store.max_amount) // Already full, no need to run a check
+			to_chat(robot, "[mat_store] could not be filled due to it already being full.")
```

Comment:
You forgot the spans here and below

Reply:
Whoops, will add them in a bit

Reply:
Added the span to this to_chat, the others imo don't per se need one, but I can add them if you insist